aviary/validation_cases/benchmark_tests/test_bench_GwGm.py

```diff
@@ -65,6 +65,30 @@ def test_bench_GwGm_SNOPT(self):
         assert_near_equal(prob.get_val("traj.desc2.timeseries.distance")[-1],
                           3675.0, tolerance=rtol)
 
+    @require_pyoptsparse(optimizer="SNOPT")
+    def test_bench_GwGm_SNOPT_lbm_s(self):
+        local_phase_info = deepcopy(phase_info)
+        prob = run_aviary('models/test_aircraft/aircraft_for_bench_GwGm_lbm_s.csv',
+                          local_phase_info, optimizer='SNOPT')
+
+        rtol = 0.01
+
+        # There are no truth values for these.
+        assert_near_equal(prob.get_val(Mission.Design.GROSS_MASS),
+                          174039., tolerance=rtol)
+
+        assert_near_equal(prob.get_val(Aircraft.Design.OPERATING_MASS),
+                          95509, tolerance=rtol)
+
+        assert_near_equal(prob.get_val(Mission.Summary.TOTAL_FUEL_MASS),
+                          42529., tolerance=rtol)
+
+        assert_near_equal(prob.get_val('landing.' + Mission.Landing.GROUND_DISTANCE),
```

Comment:
These are fine for now, I'll need to update the names in PR 149 assuming this gets pulled in first.